src/ui/viewmodels/TriggerViewModel.cpp

```diff
@@ -35,6 +35,7 @@ TriggerViewModel::TriggerViewModel() noexcept
     m_vConditionTypes.Add(ra::etoi(TriggerConditionType::SubSource), L"Sub Source");
     m_vConditionTypes.Add(ra::etoi(TriggerConditionType::AddHits), L"Add Hits");
     m_vConditionTypes.Add(ra::etoi(TriggerConditionType::SubHits), L"Sub Hits");
+    m_vConditionTypes.Add(ra::etoi(TriggerConditionType::Remember), L"Remember");
```

Comment:
I think `Remember` should be lower on the list - maybe between `Or Next` and `Measured`, or all the way down after `Trigger.